forest/src/cli/wallet_cmd.rs

```diff
@@ -0,0 +1,208 @@
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
+                println!("{}", response);
+            }
+            Self::Import { key } => {
+                use std::str;
+                let decoded_key_result = hex::decode(key);
+
+                if decoded_key_result.is_err() {
+                    cli_error_and_die("Key must be hex encoded", 1);
+                }
+
+                let decoded_key = decoded_key_result.unwrap();
+
+                let key_str = str::from_utf8(&decoded_key).unwrap();
+
+                let key_result: Result<KeyInfoJson, serde_json::error::Error> =
+                    serde_json::from_str(&key_str);
+
+                if key_result.is_err() {
+                    cli_error_and_die(&format!("{} is not a valid key to import", key), 1);
+                }
+
+                let key = key_result.unwrap();
+
+                let _ = wallet_ops::wallet_import(key.0)
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
+            }
+            Self::List => {
+                let response = wallet_ops::wallet_list()
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
+
+                response.iter().for_each(|address| {
+                    println!("{}", address.0);
+                });
+            }
+            Self::SetDefault { key } => {
+                let key = key.parse().unwrap();
+                wallet_ops::wallet_set_default(key)
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
+            }
+            Self::Sign { address, message } => {
+                let address_result = Address::from_str(address);
+
+                if address_result.is_err() {
+                    cli_error_and_die(&format!("{} is not a valid address", address), 1);
+                }
+
+                let address = address_result.unwrap();
+
+                let message = hex::decode(message).unwrap();
+                let message = base64::encode(message);
+
+                let response = wallet_ops::wallet_sign(address, message.into_bytes())
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
+                println!("{}", hex::encode(response.0.bytes()));
+            }
+            Self::Verify {
+                message,
+                address,
+                signature,
+            } => {
+                let sig_bytes = hex::decode(signature).unwrap();
+                let signature = match address.chars().nth(1).unwrap() {
+                    '1' => Signature::new_secp256k1(sig_bytes),
+                    '3' => Signature::new_bls(sig_bytes),
+                    _ => {
+                        println!("unimplemented signature type (must be bls or secp256k1)");
```

Comment:
`eprintln!()` and let's print the value maybe

Reply:
Made to use `cli_error_and_die`